test/CadlRanchProjects/authentication/http/custom/src/Generated/AuthenticationHttpCustomModelFactory.cs

```diff
@@ -1,16 +0,0 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License.
-
-// <auto-generated/>
-
-#nullable disable
```

Comment:
Yes. You could see the changes in regen.
We have more cases when we have public serialization because we have more entries in model factory with the feature on.